site/components/Modal/index.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { useEffect, FC } from "react";
+import styled from "@emotion/styled";
+import Close from "@mui/icons-material/Close";
+import { Text } from "@klimadao/lib/components";
+
+import * as styles from "./styles";
+
+type ModalWrapperProps = {
+  showModal: boolean;
+};
+
+export const ModalWrapper = styled.div`
+  display: ${(props: ModalWrapperProps) =>
```

Comment:
oh gross, another reason I left styled-components 😛 😛 😛 

keeping logic in the react/js scope, and keeping classnames & styles in the stylesheets, is more readable and predictable IMO
```
// js
className={cx({ hidden: !props.showModal })}

// css
.hidden {
  display: none;
}
```